test-variables.yml.example

```diff
@@ -3,6 +3,7 @@ BROWSER:
   # List of Chrome options - https://peter.sh/experiments/chromium-command-line-switches/
   # --disable-dev-shm-usage  and --no-sandbox are required for running chromedriver in a container
   OPTIONS: add_argument("--ignore-certificate-errors");add_argument("window-size=1920,1024");add_argument("--disable-dev-shm-usage");add_argument("--no-sandbox")
+OSD_CLUSTERMGMT_URL: "https://console.redhat.com/openshift/details/s/SoMeClUsTeRiDhAsH"
```

Comment:
to add a key-value in the test-variables you may need to apply a similar change to utils/testconfig/generateTestConfigFile and utils/testconfig/test-variables.yml.